### DISCUSSION TITLE
src\main\index.js文件里引用 require('./folder/file.js'), 运行dev，提示找不到module

### DISCUSSION THREAD

**Discussion by lling1105** [alex8088/electron-vite#640] [Q&A]
主进程目录除了index.js还有一些业务逻辑代码，需要在主进程引用并调用，npm run dev，在out目录里只有main\index.js, 导入提示找不到./folder/file.js模块。 
这个问题可以简化为：主进程index.js里使用require引入的自定义module.export模块并没有被输出到out目录
请问是我放错位置吗？或缺少配置？理想情况，主进程目录所有被引用的业务逻辑代码应一起复制到out目录才能正常运行，
谢谢！

**Comment by alex8088** [comment]
使用es语法导入模块
